web/crux-ui/src/hooks/use-throttleing.ts

```diff
@@ -19,11 +20,14 @@ export const useThrottling = (delay: number): ((action: VoidFunction) => void) =
       clearTimeout(schedule.current)
     }
 
-    schedule.current = setTimeout(() => action.trigger(), delay)
+    schedule.current = action.immediate
+      ? setTimeout(() => action.trigger(), 0)
```

Comment:
`setTimeout(() => action.trigger(), 0)` can be `setTimeout(() => action.trigger())` imho

Reply:
I will check it ty. Futhermore I want to discuss the whole solution with @m8vago later today